### PR TITLE
CLEAR and TinyImageNet tests bugfixes

### PR DESCRIPTION
**Pull request by prabhuteja12**
A few tests for datasets fail due to modified interfaces etc. This PR fixes those.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/renate/data/test_data_module.py [hunk 1]
```diff
@@ -91,15 +91,15 @@ def test_torchvision_data_module(tmpdir, dataset_name, num_tr, num_te, x_shape):
 @pytest.mark.parametrize(
     "dataset_name,chunk_id,num_tr,num_te",
     [
-        ("CLEAR10", 0, 2986, 500),
-        ("CLEAR100", 0, 9945, 4984),
+        ("CLEAR10", 0, 3300, 550),
+        ("CLEAR100", 0, 9964, 5000),
```
Comment (wistuba) [inline test/renate/data/test_data_module.py:95]: did you verify that the first number is not 10K but 9964?

File: test/renate/data/test_data_module.py [hunk 2]
```diff
@@ -131,7 +131,7 @@ def test_tiny_imagenet_data_module(tmpdir):
     assert isinstance(val_data, Dataset)
     assert len(test_data) == num_te
     assert isinstance(test_data, Dataset)
-    assert train_data[0][0].size() == test_data[0][0].size() == (3, 64, 64)
+    assert train_data[0][0].size == test_data[0][0].size == (64, 64)
```
Comment (wistuba) [inline test/renate/data/test_data_module.py:134]: can we drop the second `[0]` and check for `(3, 64, 64)`?

File: test/renate/data/test_data_module.py (hunk 1 above)
Comment (prabhuteja12) [reply test/renate/data/test_data_module.py:95]: Yes.

File: test/renate/data/test_data_module.py (hunk 2 above)
Comment (prabhuteja12) [reply test/renate/data/test_data_module.py:134]: No. `train_data[0]` is the first dataset element, and `train_data[0][0]` is a PIL Image of size 64x64.

**Review by wistuba — APPROVED** [review]

**Pull request merged**